rpc/grpc/api.go

```diff
@@ -96,6 +96,9 @@ func (blockAPI *BlockAPI) StartNewBlockEventListener(ctx context.Context) error
 				return nil
 			}
 		case event, ok := <-blockAPI.newBlockSubscription.Out():
+			if blockAPI.env.ConsensusReactor.WaitSync() {
+				continue
```

Comment:
nitpick: log warning message

Reply:
A warning will keep showing for every block added during catchup, and that's not ideal. Maybe we should find a different way to let the user know that the node is still catching up and therefore won't be able to supply new blocks events